PocketKit/Sources/Sync/Operations/FetchList.swift

```diff
@@ -77,7 +77,7 @@ class FetchList: SyncOperation {
 
     private func fetchTags() async throws {
         var shouldFetchNextPage = true
-        var pagination = PaginationInput()
+        var pagination = PaginationInput(first: .null)
```

Comment:
someone correct me if im wrong, but is `first: .null` just setting the first edge of the page?

Reply:
nvm, sorry. this seems fine. this returns the `first _n_ elements`. 
in that case, do we need this paginated if we are just grabbing every single tag?

Reply:
By default, it returns 30 for each page (set by backend). I think pagination is still useful to save on network response time and payload size. If 30 is the best number for retrieving tags, not sure, but might be a discussion that we will want to have with backend.

Reply:
@timc-mozilla 
> in that case, do we need this paginated if we are just grabbing every single tag?

Yes, there are some users with thousands of tags we'd need to look at analytics to determine our max use case.